boiler/commands/Admin/eval.js

```diff
@@ -0,0 +1,55 @@
+const { Command } = require('../../../lib')
+const adminOnly = require('../../permissions/admin')
+const { inspect } = require('util')
+
+module.exports = new Command({
+  name: 'eval',
+  description: 'Evaluate Javascript Code',
+  aliases: [ 'ev' ],
+  options: {
+    middleware: adminOnly,
+    parameters: [ 'code' ]
+  },
+  run: async ({ bot, msg, params }) => {
+    let evaled
+    try {
+      const stopwatchStart = process.hrtime()
+      evaled = eval(params.join(' ')) // eslint-disable-line no-eval
+      if (evaled instanceof Promise) {
+        evaled = await evaled
+      } // So that We Can resolve the promise
+      const stopwatchEnd = process.hrtime(stopwatchStart)
+
+      let response = '' // Woow, an empty string? see below
+
+      response += `**Output:**\n\`\`\`js\n${clean(
+        inspect(evaled, { depth: 0 }),
+        bot.token
+      )}\`\`\``
+      response += `\n\n**Type:**\`\`\`${typeof evaled}\`\`\``
+      response += `\n\n\n⏱️ \`${(stopwatchEnd[0] * 1e9 + stopwatchEnd[1]) /
+        1e6}ms\``
+
+      if (response.length > 0) {
+        // To Check if response has something to send xd
+        return response
```

Comment:
if response is empty we should probably return a message stating that it had no return value.